teps/0015-pause-pipeline.md

```diff
@@ -0,0 +1,105 @@
+---
+title: pause-pipeline
+authors:
+  - "@jbarrick-mesosphere"
+creation-date: 2020-09-10
+last-updated: 2020-09-10
+status: draft
+---
+
+# TEP-0015: Support Paused PipelineRuns and TaskRuns
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Requirements](#requirements)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Platform Implementor](#platform-implementor)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This TEP proposes a method for pausing PipelineRuns and TaskRuns that are then
+ignored by the Tekton reconciler. These paused resources can be used by platform
+implementers to control PipelineRun and TaskRun resources, for example, in cases
```

Comment:
In my case, I need the `pause` status so that the user can do some manual work after a certain task has been completed. For example, once the preprocessing task is complete, we'd like to pause the pipeline, do some label work manually, and after that we can cancel the `pause` status and let the pipeline run.

Reply:
My case is quite different from @jbarrick-mesosphere 's and my PR is based on this "manual work needed" situation 🤔